rockets/http/connection.cpp

```diff
@@ -131,8 +131,10 @@ int Connection::writeResponseBody()
 {
     if (!responseHeadersSent)
         throw headers_not_sent_error;
+#if LWS_LIBRARY_VERSION_NUMBER >= 2000000
     if (responseBodySent)
         throw body_already_sent_error;
+#endif
```

Comment:
I think it should be:
```
     if (responseBodySent)
#if LWS_LIBRARY_VERSION_NUMBER >= 2000000
         throw body_already_sent_error;
#else
         return;
#endif
```
Otherwise the body will be sent twice

Reply:
No, seems necessary. Body is split into multiple parts?